CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ The format is based on [Keep a Changelog] and this project adheres to [Semantic
   [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
   [Semantic Versioning]: http://semver.org/spec/v2.0.0.html
 
+## [0.4.1] - 2023-01-25
```

Comment:
**fix**: It looks like with me and 0.4.0, we became inconsistent with these titles. Could you make this a link and remove the link further below. We can fix the 0.4.0 lack of link in this PR or another.

Reply:
Fixed the link on `v0.4.0` so that all version titles are links
<img width="1122" alt="Screenshot 2023-01-25 at 16 35 26" src="https://user-images.githubusercontent.com/15250704/214621858-4c85843b-82ee-49d6-bdcf-1a0505e4bdbf.png">

Reply:
Thanks!